data/transport missions.txt

```diff
@@ -3404,3 +3404,69 @@ mission "Capture Smuggler"
 		payment 150000
 		conversation
 			`The huge man drags the unconscious smuggler out of the cargo container with one hand and carries the cargo box with the other. He stops in front of you. "<first> <last>," he says. "My employer, he will be very happy. Your payment, you will receive soon. Is it not good, to be reliable?" The man leaves your ship, letting the smuggler's legs scrape along the ground as he walks. A moment later, you receive <payment> from an anonymous account.`
+
+
+
+mission "Inheritance Redirection"
+	description "Travel to <destination> to act as heir for the late warlord Limping Pappa."
+	source
+		government "Pirate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		"net worth" >= 2000000
```

Comment:
wow, that's cruel. imagine hitting this with just over 200k. granted, it's your own fault (and a super obvious scheme on top of that), but it might still be nice to raise this to 300k. Perhaps i just want to see what happens, but only have 200k?

Reply:
I'm cruel, but not that cruel. :-) This number here is 2000k.

I didn't find a way to check the actual amount of credits the player has. If such a check exists, then it is fine for me to check if the player has at least 300k actual credits.

Reply:
There apparently is a credits check. I'll add it for this mission as well.

Reply:
Players now lose only half of the credits they own or less.